plans/DelegatedSigning.md

```diff
@@ -0,0 +1,95 @@
+# DelegatedSigning API
+
+This is an API to help wallets implement SEP-7 support:
+https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0007.md
+
+TODO: Use this as a guide for a ref implementation:
+https://gist.github.com/nikhilsaraf/ff3ae46116b6ae6dbdcd1743ad9495ec
+
+## API surface
+
+```typescript
```

Comment:
from methods below it may be useful to have a `setHorizonURL` method on this API to be used by the SDK internally for any calls it makes to horizon.